Java/Vishal/ValidParentheses.java

```diff
@@ -0,0 +1,70 @@
+/*
+  Problem - Valid Parentheses
+  
+  Given a string s containing just the characters '(', ')', '{', '}', '[' and ']', determine if the input string is valid.
+  
+  An input string is valid if:
+  
+    - Open brackets must be closed by the same type of brackets.
+    - Open brackets must be closed in the correct order.
+    - Every close bracket has a corresponding open bracket of the same type.
+     
+  
+  - Example 1:
+      Input: s = "()"
+      Output: true
+  
+  - Example 2:
+      Input: s = "()[]{}"
+      Output: true
+      
+  - Example 3:
+      Input: s = "(]"
+      Output: false
+       
+  
+  **Constraints**:
+  
+  1 <= s.length <= 104
```

Comment:
this conditional check is missing

Reply:
Constraints are the guidelines. This line means that given string will be of length between 1- 104.

Reply:
oh okay, my bad